src/components/bottom-sheet/send-transaction-sheet/send-transaction-sheet.tsx

```diff
@@ -51,16 +55,32 @@ export class LedgerMessageBottomSheetComponent extends React.Component<
 
         return (
             <View style={[styles.content, { height: this.props.snapPoints.bottomSheetHeight }]}>
-                <Text key="message" style={styles.message}>
-                    {this.props.reviewTransaction === true
-                        ? translate('Send.reviewTransaction')
-                        : translate('CreateHardwareWallet.openApp', {
-                              app: this.props.blockchain
-                          })}
-                </Text>
-                <View key="loading">
-                    <LoadingIndicator />
-                </View>
+                {this.props.sendTransactionMessage !== undefined && (
+                    <Text
+                        key="message"
+                        style={[
+                            styles.message,
+                            this.props.sendTransactionMessage.type === TransactionMessageType.ERROR
```

Comment:
hint: if you don't need the else branch you can use `&&`. Ex:
```
this.props.sendTransactionMessage.type === TransactionMessageType.ERROR && styles.messageError
```
you can leave it like this for now, it's just a hint :)